modules/security-understanding-container-validation.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-understanding.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="security-understanding-container-validation_{context}"]
+= Container security and validation
+
+{product-title} is released as a set of container images that comprise the different functions of the software.
+These images must be validated in order to ensure that they have not been tampered with by an external party, and to ensure that they are the intended images provided by Red{nbsp}Hat.
+
+[id="security-understanding-image-integrity_{context}"]
+== Image integrity validation at pull time
+
+When images are downloaded, or pulled, from a registry, an {product-title} cluster performs the following actions to ensure the integrity of each image:
+
+. The cluster downloads an image manifest and calculates its digest.
+
+. The cluster compares the calculated digest of the image manifest with the known digest.
+If the digest values do not match, the image is rejected without further processing.
+
+. The cluster downloads an image config file, which contains metadata such as SHA256 digests for each part of the container image, and calculates the digest of the image config file.
```

Comment:
I think it would be fair to trim out unnecessary details out. I think I added that part in to contextualize what an image config file even is, so it wouldn't just say "there's an image config file and that's part of the validation".

How about I change this line to "The cluster downloads an image config file and calculates its digest." to keep it simple?

Reply:
Nothing else in the article specifically discusses the image config, so I’m not sure it’s necessary to contextualize it.

… but I wouldn’t know what customers are asking about.